Sources/Controllers/MessagesViewController+Keyboard.swift

```diff
@@ -81,23 +92,32 @@ extension MessagesViewController {
         }
     }
 
-    // MARK: - Helpers
+    private func computeScrollViewBottomInset(forKeyboardFrame keyboardFrame: CGRect) -> CGFloat {
```

Comment:
For consistency with other functions in the project, could this function name be changed to `requiredScrollViewBottomInset(for keybaordFrame: CGRect) -> CGFloat`